homeassistant/components/philips_js/config_flow.py

```diff
@@ -0,0 +1,83 @@
+"""Config flow for Philips TV integration."""
+import logging
+
+from haphilipsjs import ConnectionFailure, PhilipsTV
+import voluptuous as vol
+
+from homeassistant import config_entries, core, exceptions
+from homeassistant.const import CONF_API_VERSION, CONF_HOST
+
+from .const import DOMAIN  # pylint:disable=unused-import
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def validate_input(hass: core.HomeAssistant, data):
+    """Validate the user input allows us to connect.
+
+    Data has the keys from STEP_USER_DATA_SCHEMA with values provided by the user.
```

Comment:
Comment is out of date: STEP_USER_DATA_SCHEMA isn't a defined constant. Instead you've defined it inline later in this file.